### PR TITLE
docs(CONTRIBUTING): commit convention details

### LINKED ISSUE #953: Commit convention details
**What would you like to be added**:

We started using conventional commits as a standard for commits in the falco repo.
We want to document that.
https://www.conventionalcommits.org/en/v1.0.0/
**Why is this needed**:

So that every contributor knows how to commit the right way while contributing to Falco.

### PR DESCRIPTION
**Pull request by fntlnz**
Signed-off-by: Lorenzo Fontana <lo@linux.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #953

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by fntlnz** [comment]
@JPLachance FYI

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,6 +98,29 @@ Once your reviewer is happy, they will say `/lgtm` which will apply the
...
 labels, does not have any `do-not-merge/*` labels, and all status checks (eg., rebase, tests, DCO) are positive.
 
+### Commit convention
+
+As commit convention, we adopt [Conventional Commits v1.0.0](https://www.conventionalcommits.org/en/v1.0.0/), we have an history
+of commits that do not adopt the convention but any new commit must follow it to be eligible for merge.
+
+#### `rule:` type
+
+Besides the classic types, we adopt a type for rules, `rule(<scope>):`.
```
Comment (leodido) [inline CONTRIBUTING.md]:
```suggestion
Besides the classic types, we adopt a prefix for rules, `rule(<scope>):`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,8 @@
       - [More about labels](#more-about-labels)
     - [Slack](#slack)
   - [Pull Requests](#pull-requests)
+    - [Commit convention](#commit-convention)
+      - [`rule:` type](#rule-type)
```
Comment (leodido) [inline CONTRIBUTING.md]:
```suggestion
      - [Rule type](#rule-type)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -96,6 +98,29 @@ Once your reviewer is happy, they will say `/lgtm` which will apply the
 Your PR will be automatically merged once it has the `lgtm` and `approved`
 labels, does not have any `do-not-merge/*` labels, and all status checks (eg., rebase, tests, DCO) are positive.
 
+### Commit convention
+
+As commit convention, we adopt [Conventional Commits v1.0.0](https://www.conventionalcommits.org/en/v1.0.0/), we have an history
+of commits that do not adopt the convention but any new commit must follow it to be eligible for merge.
+
+#### `rule:` type
```
Comment (leodido) [inline CONTRIBUTING.md]:
```suggestion
#### Rule type
```

**Pull request merged**